src/styles/stars-styles.css

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(images/star.svg) no-repeat center;
+}
+
+.stars__star:not(:nth-child(n+5)) {
+  margin-right: 4px;
+  background-image: url(images/star-active.svg);
+}
```

Comment:
reuse all styles from task with stars

Reply:
Don't completely understand what do you mean by 'reuse all styles from task with stars".I've dragged all styles used for specific block(the one i use in current task).